app/src/main/java/app/revanced/integrations/music/patches/player/PlayerPatch.java

```diff
@@ -136,6 +137,16 @@ public static void hideAudioVideoSwitchToggle(View view) {
         }
     }
 
+    public static void hideAudioVideoSwitchToggleFor0620(View view) {
+        if (!Settings.HIDE_AUDIO_VIDEO_SWITCH_TOGGLE.get())
+            return;
+
+        if (view.getParent() instanceof ViewGroup viewGroup) {
+            // removeView causes a crash on 6.20
+            viewGroup.setVisibility(View.INVISIBLE);
```

Comment:
Really?
I tried it, but it doesn't work on my side... (6.20.51 & 7.17.51)
Toggle switch still remain when I use `view.setVisibility(View.INVISIBLE)`.

```
public static void hideAudioVideoSwitchToggle(View view) {
    if (!Settings.HIDE_AUDIO_VIDEO_SWITCH_TOGGLE.get())
        return;

    view.setVisibility(View.INVISIBLE);
}
```

Here is my results

```
1: viewGroup.removeView(view); // Causes crash on 6.20.51
2: viewGroup.setVisibility(View.GONE); // Layout is broken
3: viewGroup.setVisibility(View.INVISIBLE); // Works
4: view.setVisibility(View.GONE); // Nothing changed
5: view.setVisibility(View.INVISIBLE); // Nothing changed
```

Should I just replace viewGroup.removeView(View) with viewGroup.setVisibility(View.INVISIBLE), instead of creating a new method?

Reply:
yeah I meant, viewGroup.setVisibility(View.INVISIBLE)